src/contracts/interfaces/IEigenPod.sol

```diff
@@ -49,13 +49,25 @@ interface IEigenPod {
         int256 sharesDeltaGwei;
     }
 
+    struct PartialWithdrawalProofRequest{
+        uint64 startTimestamp;
+        uint64 endTimestamp;
+        address recipient;
+        REQUEST_STATUS status;
+    }
 
     enum PARTIAL_WITHDRAWAL_CLAIM_STATUS {
         REDEEMED,
         PENDING,
         FAILED
     }
 
+     enum REQUEST_STATUS {
+        PENDING, // request is pending
```

Comment:
default status probably shouldn't be 'PENDING'

Reply:
ah good point